cis/publisher.py

```diff
@@ -162,13 +183,44 @@ def _reintegrate_profile_with_api(self):
                 if group.split('_')[0] != self.publisher.get('id'):
                     reintegrated_groups.append(group)
 
-            logger.debug('Groups successfully reintegrated. Replacing group list in proposed profile.')
+            logger.info(
+                'Groups successfully reintegrated for user_id {} for change request from publisher {}.'.format(
+                    self.profile_data.get('user_id'),
+                    self.publisher.get('id')
+                )
+            )
+
+            self.stats(vault_profile, self.profile_data)
             # Replace the data in the profile with our reintegrated group list.
             self.profile_data['groups'] = reintegrated_groups
 
-    def _generate_signature(self):
+    def stats(self, current_user_profile, proposed_change):
```

Comment:
what is this used for?

Reply:
It generates some log messages about how many groups were added or removed.  I'm going to turn this into a cloudwatch metric for mass group add / drops for anomalies.

Reply:
oh cool